packages/mobile/src/screens/account-screen/AccountDrawer.tsx

```diff
@@ -206,7 +206,7 @@ export const AccountDrawer = (props: AccountDrawerProps) => {
             width={spacing(7)}
           />
           <Text fontSize='large' weight='heavy'>
-            {totalBalance ? formatWei(totalBalance, true) : 0}
+            {totalBalance ? formatWei(totalBalance, true, 0) : 0}
```

Comment:
thank u

Reply:
love that you have to add a boolean and a zero to do what we want lmao